src/WebJobs.Script/Config/AppServiceOptions.cs

```diff
@@ -12,5 +12,7 @@ public class AppServiceOptions
         public string RuntimeSiteName { get; set; }
 
         public string SlotName { get; set; }
+
+        public string AzureMonitorTraceCategory { get; set; }
```

Comment:
Optional suggestion: instead of having the users of this option class call `IsAzureMonitorLoggingEnabled`, you could calculate and set it as a bool directly on this object.

``` CSharp
// AppServiceOptions.cs
public bool IsAzureMonitorEnabled { get; set; }

// AppServiceOptionsSetup.cs
// Configure(AppServiceOptions options)
options.IsAzureMonitorEnabled = _environment.IsAzureMonitorEnabled();
```